connect/src/androidTest/java/com/stripe/android/connect/ConnectTestRunner.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.connect
+
+import android.app.Application
+import android.content.Context
+import androidx.test.runner.AndroidJUnitRunner
+import com.stripe.android.connect.di.DaggerTestStripeConnectComponent
+import com.stripe.android.connect.di.StripeConnectComponent
+
+@Suppress("unused")
+class ConnectTestRunner : AndroidJUnitRunner() {
+    override fun newApplication(cl: ClassLoader?, className: String?, context: Context?): Application {
+        StripeConnectComponent.instance = DaggerTestStripeConnectComponent.create()
```

Comment:
👀 Replacing the global root dagger component feels weird, but I can't think of anything better.